### PR TITLE
Walt pool create

### PR DESCRIPTION
**Pull request by wkrapohl**
Create Hive Pool

### CODE REVIEW

File: ansible/provision-pool-play/examples/roks-pool-vars.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+# PLACE HOLDER ONLY
+"roks_api_token": "3-rKI5k7QG10Dq0ycSGiPHxb6hqfXmbeybahqshO-fI"
```
Comment (rayashworth) [inline ansible/provision-pool-play/examples/roks-pool-vars.yml:3]: did you mean to include the token?

File: ansible/provision-pool-play/requirements.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+    collections:
```
Comment (rayashworth) [inline ansible/provision-pool-play/requirements.yml]: you should be able to use the prereq role and update if need be.

File: ansible/provision-pool-play/roles [hunk 3]
```diff
@@ -0,0 +1 @@
+../roles
```
Comment (rayashworth) [inline ansible/provision-pool-play/roles:1]: we can now include these in the ansible.cfg in the playbook's parent folder.

**Review by rayashworth — APPROVED** [review]
Looks good.

commented on collections, ../roles symlink, and roks token

File: ansible/provision-pool-play/examples/roks-pool-vars.yml (hunk 1 above)
Comment (wkrapohl) [reply ansible/provision-pool-play/examples/roks-pool-vars.yml:3]: Came from copy of provision-ocp-cluster play, I did nathing to add this!

File: ansible/provision-pool-play/roles (hunk 3 above)
Comment (wkrapohl) [reply ansible/provision-pool-play/roles:1]: Can you point to an example in the community repo that can be used?

**Pull request merged**